stdlib/curses/__init__.pyi

```diff
@@ -14,12 +14,12 @@ _T = TypeVar("_T")
 _P = ParamSpec("_P")
 
 # available after calling `curses.initscr()`
-LINES: int
-COLS: int
+LINES: Final[int]
+COLS: Final[int]
 
 # available after calling `curses.start_color()`
-COLORS: int
-COLOR_PAIRS: int
+COLORS: Final[int]
+COLOR_PAIRS: Final[int]
```

Comment:
are these actually `Final`? they change value throughout runtime and aren't true constants